ns-theme-check.php

```diff
@@ -27,3 +27,18 @@
 
 // Load admin.
 require_once NS_THEME_CHECK_DIR . '/inc/admin.php';
+
+/**
+ * Add go to theme check page link on plugin page.
+ *
+ * @since 0.1.2
+ */
+
+function ns_theme_check_plugin_settings_link( $links ) { 
+  $settings_link = '<a href="themes.php?page=ns-theme-check">' . esc_attr( 'Theme Check Page', 'ns-theme-check' ) . '</a>'; 
```

Comment:
Would be better to use tabs not spaces. I would name the variable `$theme_check_link`.